doc/performance.md

```diff
@@ -152,3 +155,14 @@ On Intel and AMD Windows platforms, Microsoft Visual Studio enables programmers
 We do not recommend that you compile simdjson with architecture-specific flags such as  `arch:AVX2`. The simdjson library automatically selects the best execution kernel at runtime.
 
 Recent versions of Microsoft Visual Studio on Windows provides support for the LLVM Clang compiler. You  only need to install the "Clang compiler" optional component. You may also get a copy of the 64-bit LLVM CLang compiler for [Windows directly from LLVM](https://releases.llvm.org/download.html). The simdjson library fully supports the LLVM Clang compiler under Windows. In fact, you may get better performance out of simdjson with the LLVM Clang compiler than with the regular Visual Studio compiler.
+
+
+Downclocking
+--------------
+
+On some Intel processors, using SIMD instructions in a sustained manner on the same CPU core may result in a phenomenon called downclocking whereas the processor initially runs these instructions at a slow speed before reducing the frequency of the core for a short time (milliseconds). Intel refers to these states as licenses. On some current Intel processors, it occurs under two scenarios:
```

Comment:
It's fine to check in as is, but the headline is buried in the second sentence of the last paragraph right now :) I'd actually put a first sentence or paragraph here with the headline "There is no downclocking due to simdjson" or something. Perhaps with appropriate caveats like "in our testing" or something.

I don't know much about the "on recent processors" bit and whether that contains a missing worry, but people will also take that and decide that there must be downclocking on older processors due to simdjson :)

Reply:
Fixed. I qualify "recent processors" because I don't want to do processor archeology into the various particularities of older processors. It is not a problem if you have an up-to-date processors.

Reply:
Cool. What does recent mean to you? When I read the word I think "I hope he means in the past 5-10 years, but he might mean processors in the last 6 months, now I'm anxious about using this." And a sentence saying "We haven't tested on processors older than X, but nothing has specifically come to our attention about it, either."

Anyway, I can go in and change/add this, too, don't feel like you have to get lost in uncounted revisions. Just trying to act the part of the fresh-eyed reader.